perf/flame.jl

```diff
@@ -17,7 +17,7 @@ run_name_list =
     ["default_modular_unthreaded", "coarse_single_modular", "target_amip_n32_shortrun", "target_amip_n1_shortrun"]
 run_name = run_name_list[parse(Int, ARGS[2])]
 allocs_limit = Dict()
-allocs_limit["perf_default_modular_unthreaded"] = 6113200
+allocs_limit["perf_default_modular_unthreaded"] = 8638304
```

Comment:
Why do we need to increase the allocation limit? Does the new way of calculating fluxes allocate more, or is this from upstream ClimaAtmos changes?

Reply:
In this case, it's expected because we're increasing the cache of the surface models, so they can store their own `q_sfc` and `rho_sfc`. I'm sure there are some potential alloc gains nevertheless though - Valeria and Charlie have already started looking into this [here](https://github.com/CliMA/ClimaCoupler.jl/issues/323).